### PR TITLE
netperf: bunch of build cleanups to get it to build cleanly

### PR DESCRIPTION
**Pull request by muliby-lb**
@sashas-lb I need these changes to get it to build cleanly...
@sagi-lb FYI

### CODE REVIEW

**Comment by sashas-lb** [comment]
@muliby-lb  please rebase on my PR

**Comment by sashas-lb** [comment]
forget it .. you are rebased

File: Makefile.am [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
 AUTOMAKE_OPTIONS = dist-bzip2 dist-zip
-SUBDIRS = src doc
+SUBDIRS = src
```
Comment (sashas-lb) [inline Makefile.am:3]: why ?

Comment (muliby-lb) [reply Makefile.am:3]: see the commit msg, building doc requires makeinfo (from texinfo) which we do not have in dockerize. It doesn't seem reasonable to install texinfo just to build netperf doc, so let's nuke it.

**Comment by muliby-lb** [comment]
@sashas-lb any other comments or can I push?

File: Makefile.am (hunk 1 above)
Comment (sashas-lb) [reply Makefile.am:3]: mm it worked for me .. strange ..

File: Makefile.in [hunk 2]
```diff
@@ -58,8 +58,8 @@ DIST_COMMON = README $(am__configure_deps) $(srcdir)/Makefile.am \
 	COPYING ChangeLog INSTALL NEWS config.guess config.sub depcomp \
 	install-sh missing mkinstalldirs
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
-am__aclocal_m4_deps = $(top_srcdir)/src/missing/m4/salen.m4 \
-	$(top_srcdir)/acinclude.m4 $(top_srcdir)/configure.ac
+am__aclocal_m4_deps = $(top_srcdir)/acinclude.m4 \
```
Comment (sashas-lb) [inline Makefile.in:61]: again whu .. i sucesfully compiled it in docker

File: aclocal.m4 [hunk 3]
```diff
@@ -981,5 +981,4 @@ AC_SUBST([am__tar])
 AC_SUBST([am__untar])
 ]) # _AM_PROG_TAR
 
-m4_include([src/missing/m4/salen.m4])
```
Comment (sashas-lb) [inline aclocal.m4:984]: same here .. i compiled it without an issue in docker

Comment (muliby-lb) [reply aclocal.m4:984]: Try on a fresh checkout. If you succeed, we have a problem with the docker.

File: configure [hunk 4]
```diff
@@ -1052,6 +1054,15 @@ do
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
```
Comment (sashas-lb) [inline configure:1057]: what is this ?

File: configure [hunk 5]
```diff
@@ -4752,7 +4764,7 @@ else
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
```
Comment (sashas-lb) [inline configure:4767]: chto?

Comment (muliby-lb) [inline configure:4767]: autogenerated file :-)

**Comment by muliby-lb** [comment]
Closing and will reopen with the relevant commits

**Pull request closed without merging**